source/grid/tria.cc

```diff
@@ -8995,28 +8995,79 @@ Triangulation (const Triangulation<dim, spacedim> &other)
 
 
 
+#ifdef DEAL_II_WITH_CXX11
+template <int dim, int spacedim>
+Triangulation<dim, spacedim>::
+Triangulation (Triangulation<dim, spacedim> &&tria)
+  :
+  Subscriptor(static_cast<Subscriptor&&>(tria)),
+  smooth_grid(tria.smooth_grid),
+  periodic_face_pairs_level_0(tria.periodic_face_pairs_level_0),
+  periodic_face_map(tria.periodic_face_map),
+  levels(tria.levels),
+  faces(tria.faces),
+  vertices(std::move(tria.vertices)),
+  vertices_used(tria.vertices_used),
```

Comment:
Ah, yes, good find. The second answer is what I was needing. I guess that means that in member initializer lists, one should generally use `std::move` for everything that's not a scalar. Can you adjust your patch accordingly?

(Incidentally, I think we should also look at the move constructors for other classes that have recently obtained move constructors.)